apps/redi-connect/src/services/api/api.tsx

```diff
@@ -51,6 +51,10 @@ export const logout = () => {
   history.push('/front/home')
 }
 
+export const entraLoginUrl = (): string => {
+  return `${NEST_API_URL}/auth/entra-id`
+}
+
```

Comment:
I'm not sure if this belongs here, as this doesn't seem to be an API call. Are there any plans for this function in the future or is it only to return this URL? I don't see any issues with using  `${NEST_API_URL}/auth/entra-id` in Line 87 of `apps/redi-connect/src/routes/routes__logged-out.tsx`.

Reply:
Right, sorry. I thought at first it needed to be used in multiple places, but it can just be set in `routes__logged-out`. ✅

Reply:
Actually since it'll eventually be used in both apps, we can define it in `libs/shared-config/src/lib/config.ts`?